lib/calculate.h

```diff
@@ -111,6 +111,10 @@ R calculate(const std::string& s, const T& x, const T& y, bool* error = nullptr)
         return wrap(x >= y);
     case '<=':
         return wrap(x <= y);
+    case '<=>':
+        if (error)
+            *error = true;
+        return R{};
```

Comment:
I think this can be calculated as `return wrap(x - y)`?

Reply:
Shouldn't that be `wrap(x < y)`?

Reply:
No, the operator works like this:

* `(a <=> b) < 0` if `lhs < rhs`
* `(a <=> b) > 0` if `lhs > rhs`
* `(a <=> b) == 0` if `lhs` and `rhs` are equal/equivalent.

So `x - y` should work for arithmetic types(which is what `calculate` uses).

Reply:
Thanks, fixed.